stable/mariadb/templates/deployment.yaml

```diff
@@ -10,6 +10,10 @@ metadata:
 spec:
   template:
     metadata:
+      {{- if and (.Values.tolerations) (le .Capabilities.KubeVersion.Minor "5") }}
```

Comment:
IMO we can remove this, we only really support Kubernetes N and N-1 (1.7 and 1.8 today).